### PR TITLE
Added checkbox for important tasks in default configuration

### PR DESCRIPTION
**Pull request by david-0609**
Screenshot_20240507_192116
Many people use Obsidian to organise their tasks using checkboxes and I added a checkbox to the default configuration that indicates that the task is important. I also added a highlight group ("ObsidianImportant") that uses a darker red colour. Also I shifted the order for toggle checkboxes to make this checkbox make sense in terms of order.

### CODE REVIEW

**Review by epwalsh — COMMENTED** [review]
This looks good, thanks @david-0609. I actually already have something similar in my personal config. Can you just update the config example in the README? Then we'll get this merged.

**Comment by david-0609** [comment]
> This looks good, thanks @david-0609. I actually already have something similar in my personal config. Can you just update the config example in the README? Then we'll get this merged.

I updated the example config in the README, you can merge now

File: README.md [hunk 1]
```diff
@@ -463,6 +463,7 @@ This is a complete list of all of the options that can be passed to `require("ob
       ["x"] = { char = "", hl_group = "ObsidianDone" },
       [">"] = { char = "", hl_group = "ObsidianRightArrow" },
       ["~"] = { char = "󰰱", hl_group = "ObsidianTilde" },
+      ["!"] = { char = "", hl_group = "ObsidianImportant" },
```
Comment (epwalsh) [inline README.md:466]: This HL group also needs to be defined below.

Comment (david-0609) [reply README.md:466]: HL Group has been updated in readme

**Review by epwalsh — APPROVED** [review]
Thanks @david-0609!

**Pull request merged**